docs/_ug/commandSummary/itemCommandsExamples/find.md

```diff
@@ -0,0 +1,2 @@
+<!-- markdownlint-disable-file first-line-h1 -->
+`find Potato`
```

Comment:
Does capitalisation matter?

Reply:
Not really, but Potato is an item name, so I capitalized it!